jaxrs-api/src/main/java/jakarta/ws/rs/GatewayTimeoutException.java

```diff
@@ -0,0 +1,105 @@
+/********************************************************************************
+ * Copyright (c) 2019 Jeyvison Nascimento
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ ********************************************************************************/
+
+package jakarta.ws.rs;
+
+import jakarta.ws.rs.core.Response;
+
+/**
+ * A runtime exception indicating that the server, while acting as a gateway or proxy,
+ * received an invalid response from the upstream server it accessed in attempting to fulfill the request.
```

Comment:
This description is identical to the one for BadGatewayException. Shouldn't it mention "timeout"?